target-app/Chat/Views/ChatsViewController.swift

```diff
@@ -19,38 +26,61 @@ class ChatsViewController: UIViewController {
         return tableView
     }()
     
-    //TODO: remove mock
-    var data: [Match] = [
-        Match(matchID: 0, topicIcon: "ic_topic_travel", lastMessage: "¡Hola! A dónde querés viajar?", unreadMessages: 0, user: User(id: 0 ,firstName: "name", lastName: "last", fullName: "José Gazzano", name: "name", username: "user", email: "email", gender: "gender", password: nil, passwordConfirmation: nil, avatar: User.Avatar(smallThumbUrl: "avatar_mock_1"))),
-        Match(matchID: 1, topicIcon: "ic_topic_movie", lastMessage: "¿Alguna película para recomendar?", unreadMessages: 1, user: User(id: 1 ,firstName: "name1", lastName: "last1", fullName: "Karen Bauer", name: "name 1", username: "user 1", email: "email1", gender: "gender1", password: nil, passwordConfirmation: nil, avatar: User.Avatar(smallThumbUrl: "avatar_mock_2")))
-    ]
+    private lazy var tableViewDataSource: UITableViewDiffableDataSource<ViewControllerSection, Match> = {
+        let dataSource = UITableViewDiffableDataSource<ViewControllerSection, Match>(tableView: tableView) {
+            tableView, _, model in
+            
+            let cell = tableView.dequeueReusableCell(withIdentifier: ConversationTableViewCell.reuseIdentifier)
+            if let cell = cell as? ConversationTableViewCell {
+                cell.updateData(match: model)
+            }
+            return cell
+        }
+        
+        return dataSource
+    }()
+    
+    enum ViewControllerSection: Hashable {
+        case main
+    }
+    
+    // MARK: - Lifecycle Events
     
     override func viewDidLoad() {
         super.viewDidLoad()
         applyDefaultUIConfigs()
         configureViews()
+        
+        setupBinders()
+        viewModel.getMatchConversations()
     }
     
-    func setupTableView() {
-        view.addSubview(tableView)
-        
-        NSLayoutConstraint.activate([
-            tableView.topAnchor.constraint(equalTo: view.topAnchor),
-            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
-            tableView.leftAnchor.constraint(equalTo: view.leftAnchor),
-            tableView.rightAnchor.constraint(equalTo: view.rightAnchor)
-        ])
-                
-        tableView.dataSource = self
+    init(viewModel: ChatsViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
     
     // MARK: - Actions
-
+    
     @objc
     func tapOnMapBarButton(_ sender: Any) {
         AppNavigator.shared.navigate(to: HomeRoutes.home, with: .changeRoot)
     }
-
+    
+    // MARK: - BINDERS
+    private var cancellables = Set<AnyCancellable>()
```

Comment:
We usually group the properties of any declaration at the top section of the file

Reply:
done!